examples/usecases/transformers-next-item-prediction-with-pretrained-embeddings.ipynb

```diff
@@ -0,0 +1,5658 @@
+{
```

Comment:
<p>Can you display the input_batch?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/NVIDIA-Merlin/models/pull/1102/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='32'/>

Reply:
<p>I tried this in a live notebook and it actually doesn't look that bad, rows are getting hidden behind <code>...</code>,will make a commit with the batch showing and will make a change if need be  </p>